accounts/models/connection.py

```diff
@@ -14,4 +14,4 @@ class Connection(models.Model):
     updated_at = models.DateTimeField(auto_now=True)
 
     def __str__(self):
-        return self.name
\ No newline at end of file
+        return self.name
```

Comment:
- check if `name` exists, if not return the id of the `Connection` instance as `Connection_{id}`